tests/infra/runner.py

```diff
@@ -141,13 +144,14 @@ def run(build_directory, get_command, args):
                                 break
                         time.sleep(1)
 
-                    tx_rates.get_metrics()
-                    for remote_client in clients:
-                        remote_client.print_and_upload_result(args.label, metrics)
-                        remote_client.stop()
+                    if not args.pbft:
```

Comment:
We still need this for PBFT so that we can track performance for it. IIRC, it was just the call to CCF's `getMetrics` that was problematic. I believe we can disable that call for now (and ticket it) but the rest should stay (the test only checks the outputs of the perf client and record that in Azure Blob Storage + Cosmos).

Reply:
This runs into an issue of large messages. I added a comment and this ticket https://github.com/microsoft/CCF/issues/534